dev/Makefile

```diff
@@ -32,12 +32,12 @@ all: create-cluster init deploy-services deploy
 
 
 redeploy-bitcoin:
-	kubectl delete namespace galoy-dev-bitcoin --force
+	-kubectl delete namespace galoy-dev-bitcoin --force
```

Comment:
What is this syntax? Never seen it before

Reply:
It makes that command allowed to fail and the Makefile to continue. 
In case the redeploy was run and failed before the namespace is already deleted so the command is unusable and need to run the `tf apply ...` manually
a `-` prefix or `|| true` on the end allows it to continue.